api/src/pcapi/routes/backoffice_v3/forms/offerer.py

```diff
@@ -76,7 +76,9 @@ class OffererValidationListForm(utils.PCForm):
     class Meta:
         csrf = False
 
-    q = fields.PCOptSearchField("Nom de structure, SIREN, code postal, département, ville, email, nom de compte pro")
+    q = fields.PCOptSearchField(
+        "Nom de structure, SIREN, code postal, dép., ville, email, nom de compte pro, id DMS CB"
```

Comment:
```suggestion
        "Nom de structure, SIREN, code postal, dép., ville, email, nom de compte pro, ID DMS CB"
```
par cohérence :rofl: